src/App.js

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
+  constructor() {
+    super();
+    this.state = {
+      
+      TodoArray: [
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: false
+        }
+      ],
+      inputText: ''
+    }
+    this.submitHandler = this.submitHandler.bind(this);
+    this.HandleChanges = this.HandleChanges.bind(this);
+
```

Comment:
Another way of doing this is by having your `submitHandler` and `HandleChanges` methods using arrow function, such that:
````
submitHandler = e => {
  ...
}
```
This way you don't need to do the binding manually.